Formula/percona-xtrabackup.rb

```diff
@@ -62,6 +62,9 @@ def install
     share.install "share/man" if build.with? "docs"
 
     rm_rf prefix/"xtrabackup-test" # Remove unnecessary files
+    # remove conflicting libraries
```

Comment:
conflicting with mysql?

Reply:
Yep, that's right. These two files are part of mysql, but this package tries to install its own versions.